CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 ### Internal
 - Change `LoggedInUser#status` to be a forward-compatible enum
 
+### Fixes
+- Fix overdue list showing last updated BP, instead of last recorded BP
```

Comment:
Can we name this something similar to how we do in the commits. How about:
`Fix overdue list to show last recorded BP instead of last updated BP`

Reply:
I will do this.